pyk/src/pyk/integration_tests/test_defn.py

```diff
@@ -29,16 +20,37 @@ def _update_symbol_table(symbol_table):
 
     def test_print_configuration(self):
         # Given
-        config = KApply('<T>', [KApply('<k>', [KApply('int_;_', [KApply('_,_', [KToken('x', 'Id'), KApply('_,_', [KToken('y', 'Id'), KApply('.List{"_,_"}')])])])]), KApply('<state>', [KApply('.Map')])])
-        config_expected = '\n'.join([ '<T>'              # noqa
-                                    , '  <k>'            # noqa
-                                    , '    int x , y ;'  # noqa
-                                    , '  </k>'           # noqa
-                                    , '  <state>'        # noqa
-                                    , '    .Map'         # noqa
-                                    , '  </state>'       # noqa
-                                    , '</T>'             # noqa
-                                    ])                   # noqa
+        config = KApply(
+            '<T>',
+            [
+                KApply(
+                    '<k>',
+                    [
+                        KApply(
+                            'int_;_',
+                            [
+                                KApply(
+                                    '_,_',
+                                    [KToken('x', 'Id'), KApply('_,_', [KToken('y', 'Id'), KApply('.List{"_,_"}')])],
+                                )
+                            ],
+                        )
+                    ],
+                ),
+                KApply('<state>', [KApply('.Map')]),
+            ],
+        )
+        # fmt: off
```

Comment:
Oh no. We can't have this as the format. Can we turn off the line-breaking features of black? Formatted code should be beautiful and compact, not taking a book to read.

Reply:
I agree with you w.r.t. the highlighted instance. In such cases formatting can be turned off locally:

```python3
# fmt: off
...
# fmt: on
```

In general I wouldn't disable formatting related to line breaking though.